ontoenv/__init__.py

```diff
@@ -129,24 +129,41 @@ def resolve_uri(
         :raises Exception: [TODO:description]
         """
         uri = str(uri)
-        graph = rdflib.Graph()
         try:
-            graph.parse(uri, format=rdflib.util.guess_format(uri) or "xml")
+            if not rdflib.util.guess_format(uri):
+                # this should be rdflib.Graph.parse guess strategy
+                try:
+                    graph = rdflib.Graph()
+                    graph.parse(uri, format='ttl')
+                except: #rdflib.exceptions.ParserError or SyntaxError?:
+                    graph = rdflib.Graph()
+                    graph.parse(uri, format='xml')
+                # ...could go through the formats
+            else:
+                graph = rdflib.Graph()
+                graph.parse(uri)
```

Comment:
I get why this is here, but the real fix is to make sure that RDFlib deals with the content-type header of the downloaded file. Falling back to trying a list of formats should be even more of a last resort!

Reply:
Actually, looks like https://brickschema.org/schema/1.3/Brick works fine when you parse... it's just `rdflib.util.guess_format` that doesn't know that it's a Turtle file.

Here's what I think the logic should be:

```python
try:
  graph.parse(uri, format=rdflib.util.guess_format(uri))
except:
  for fmt in ['ttl','json-ld','xml']:
    graph = rdflib.Graph()
    try:
      graph.parse(url, format=fmt)
      break
    except:
      continue # log error too
```